packages/regl-worldview/src/Worldview.js

```diff
@@ -153,24 +153,7 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
     if (this.props.cameraState) {
       worldviewContext.cameraStore.setCameraState(this.props.cameraState);
     }
-
-    // queue up a paint operation on the next frame, if we haven't already
-    if (!this._tick) {
-      this._tick = requestAnimationFrame(() => {
-        this._tick = undefined;
-        try {
-          worldviewContext.paint();
-        } catch (error) {
-          // Regl automatically tries to reconnect when losing the canvas 3d context.
-          // We should log this error, but it's not important to throw it.
-          if (error.message === "(regl) context lost") {
```

Comment:
I think you might need to keep this somewhere, maybe @davidswinegar can advise more but I think this error was causing lots of problems for us in sentry.

Reply:
+1 to keeping this filter as we experienced the same spam on our team's error reporter.

Reply:
I'm not an expert on WebGL but this is a code smell from an error/flow control perspective. Generally when a program has an "error" and happily continues, that to me suggests something is wrong.  Is it possible we're not discerning between the "expected" error, and an actual error here? What if the reconnection code in regl actually failed, would it just throw the same error?

I'll add this logic back, in the `WorldviewContext`, for now... but can't we also just mute errors in Sentry? Or alternatively, would it make sense to send a PR to regl to not throw the error if there was a successful reconnect?

Reply:
> would it make sense to send a PR to regl to not throw the error if there was a successful reconnect

I remember us doing that, or maybe we only talked about it? I think it is a mistake in regl that should ideally be fixed there. @jtbandes @brianc do you remember if we ended up making a PR to regl?

Reply:
Or @troygibb? https://github.com/cruise-automation/webviz/pull/265

Reply:
I _asked them_ about making a PR for a [separate issue](https://github.com/regl-project/regl/issues/533) (not having an unstoppable `console.error` if you double-dispose of a texture).  I never talked w/ them about a context lost error.  That's a bit alarming to just swallow it entirely as sometimes if the context is lost...it's gone for good and yer app crashes.  Maybe regl throws the same error somewhere else if context is lost?

Reply:
Thanks for the info. Yeah agreed, but at least this decision is decoupled from my PR now.